babi/linters/pre_commit.py

```diff
@@ -66,7 +66,13 @@ def command(self, filename: str, scope: str) -> tuple[str, ...] | None:
         if not os.path.exists(os.path.join(root, '.pre-commit-config.yaml')):
             return None
 
-        return ('pre-commit', 'run', '--color=never', '--files', filename)
+        return (
+            'pre-commit',
+            'run',
+            '--config=".pre-commit-config.yaml"',
```

Comment:
the quotes here are incorrect -- this isn't a shell

also the file should be `os.path.join(root, '.pre-commit-config.yaml')` as that's what's checked above

Reply:
Aha! Lemme change that really quick.

Reply:
I am unsure if I changed it correctly. So please have a look again!